responder/events.go

```diff
@@ -96,3 +118,54 @@ func NotifyMSPlannerResponder(responder api.Responder) (string, error) {
 
 	return *task.GetId(), nil
 }
+
+func NotifyJiraResponderAddComment(responder api.Responder, comment string) (string, error) {
+	client, err := jiraClientFromResponder(responder)
+	if err != nil {
+		return "", err
+	}
+
+	commentObj, err := client.AddComment(responder.ExternalID, comment)
+	if err != nil {
+		return "", err
+	}
+
+	return commentObj.ID, nil
+}
+
+func msPlannerClientFromResponder(responder api.Responder) (msplanner.MSPlannerClient, error) {
+	if responder.Properties["responderType"] != MSPlannerResponder {
+		return msplanner.MSPlannerClient{}, fmt.Errorf("invalid responderType: %s", responder.Properties["responderType"])
+	}
+
+	teamSpecJson, err := responder.Team.Spec.MarshalJSON()
+	if err != nil {
+		return msplanner.MSPlannerClient{}, err
+	}
+
+	var teamSpec api.TeamSpec
+	if err = json.Unmarshal(teamSpecJson, &teamSpec); err != nil {
+		return msplanner.MSPlannerClient{}, err
+	}
+
+	return msplanner.NewClient(
+		teamSpec.ResponderClients.MSPlanner.TenantID,
+		teamSpec.ResponderClients.MSPlanner.ClientID,
+		teamSpec.ResponderClients.MSPlanner.GroupID,
+		teamSpec.ResponderClients.MSPlanner.Username.Value,
+		teamSpec.ResponderClients.MSPlanner.Password.Value,
+	)
+}
+
+func NotifyMSPlannerResponderAddComment(responder api.Responder, comment string) (string, error) {
+	client, err := msPlannerClientFromResponder(responder)
+	if err != nil {
+		return "", err
+	}
+
+	err = client.AddComment(responder.ExternalID, comment)
+
+	// MS Graph API does not return the ID of the comment created
+	// so we just return a constant string for now
+	return "posted", err
+}
```

Comment:
I think all of this should go into the msplanner package,  and then maybe add a registry of responderType to responder interface

Reply:
Yeah, moving it to a package is better, I'll pick up the interface part in the config task